scenarios/scenario-10/README.md

```diff
@@ -0,0 +1,116 @@
+## Scenario 10 - Apply interceptors to the API
+- This scenario describes how to apply interceptors to carry out transformations and mediations on the requests and responses.
+- First, we need to implement custom request interceptors and response interceptors. For more information about implementing custom interceptors you can refer the document [adding interceptors.](https://docs.wso2.com/display/MG300/Adding+Interceptors)
```

Comment:
We need to mention that we have provided sample interceptors. If you want to learn more about it please refer the documentation. Otherwise, it gives the idea that refering document is a must step.

Reply:
And what those interceptors would do.

Reply:
made the changes. each interceptor method described in steps